doc/API.md

```diff
@@ -202,7 +202,7 @@ axe.configure({
   - `rules` - Used to add rules to the existing set of rules, or to override the properties of existing rules
     - The rules attribute is an Array of rule objects
     - each rule object can contain the following attributes
-      - `id` - string(required). This uniquely identifies the rule. If the rule already exists, it will be overridden with any of the attributes supplied. The attributes below that are marked required, are only required for new rules.
+      - `id` - string(required). This uniquely identifies the rule. If the rule already exists, it will be overridden with any of the attributes supplied. The attributes below that are marked required, are only required for new rules. - `impact` - string(optional). Override the impact defined by checks
```

Comment:
Looks like a newline is needed before the `impact` list item. Right now it's part of the same paragraph as `id`.